playground/backend/cmd/server/controller.go

```diff
@@ -294,3 +294,28 @@ func (controller *playgroundController) GetPrecompiledObjectLogs(ctx context.Con
 	response := pb.GetPrecompiledObjectLogsResponse{Output: logs}
 	return &response, nil
 }
+
+// GetDefaultPrecompiledObject returns the default precompile object for sdk.
+func (controller *playgroundController) GetDefaultPrecompiledObject(ctx context.Context, info *pb.GetDefaultPrecompiledObjectRequest) (*pb.GetDefaultPrecompiledObjectResponse, error) {
+	switch info.Sdk {
+	case pb.Sdk_SDK_UNSPECIFIED, pb.Sdk_SDK_SCIO:
+		logger.Errorf("GetDefaultPrecompiledObject(): unimplemented sdk: %s\n", info.Sdk)
+		return nil, errors.InvalidArgumentError("Error during preparing", "Sdk is not implemented yet: %s", info.Sdk.String())
+	}
+
+	bucket := cloud_bucket.New()
+	precompiledObject, err := bucket.GetDefaultPrecompileObject(ctx, info.Sdk, controller.env.ApplicationEnvs.WorkingDir())
+	if err != nil {
+		logger.Errorf("GetDefaultPrecompileObject(): cloud storage error: %s", err.Error())
+		return nil, errors.InternalError("Error during getting default Precompiled Object", "Error with cloud connection")
+	}
+
+	response := pb.GetDefaultPrecompiledObjectResponse{PrecompiledObject: &pb.PrecompiledObject{
+		CloudPath:       precompiledObject.CloudPath,
+		Name:            precompiledObject.Name,
+		Description:     precompiledObject.Description,
+		Type:            precompiledObject.Type,
+		PipelineOptions: precompiledObject.PipelineOptions,
+	}}
```

Comment:
Shouldn't we return the code of the example as well? Or it will be requested from the frontend after receiving this response?

Reply:
Yes, this information will be enough for frontend.

Reply:
Could you please also add the `link` field according to [this one](https://github.com/apache/beam/pull/16506)